apps/notifications/src/panel/templates/gridicons.jsx

```diff
@@ -1,354 +1,349 @@
 import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
-import { Component } from 'react';
-
-export default class extends Component {
-	static displayName = 'Gridicons';
-
-	static propTypes = {
-		icon: PropTypes.string.isRequired,
-		size: PropTypes.number,
-		onClick: PropTypes.func,
+import { forwardRef } from 'react';
+
+const Gridicons = forwardRef( ( { icon, size = 24, onClick }, ref ) => {
```

Comment:
```suggestion
const Gridicons = forwardRef( ( { iconName, size = 24, onClick }, ref ) => {
```

Reply:
This is more descriptive, but I'd need to change it everywhere the component is called. For this PR I'd rather limit the scope to this file.